### PR TITLE
No table

### PR DESCRIPTION
**Pull request by FranklinTOliveira**

### CODE REVIEW

**Comment by FranklinTOliveira** [comment]
This is NoTable, an adjustment to have this web page not have a table, if without parameters.

**Comment by FranklinTOliveira** [comment]
This is NoTable, an adjustment to have this web page not have a table, if without parameters.

**Comment by FranklinTOliveira** [comment]
So I added Wickets for the headings. 

But I'm unsure of how to add whitespace in the Java. And it seems that using !=houseName doesn't read as "if no houseName" though that would seem to be the correct code for that phrase.

**Comment by FranklinTOliveira** [comment]
I think I'm on the verge of success now! But some error somewhere seems to be cutting off access to kingsService. So long as that's in play, I won't even be able to run the site on the browser.

**Comment by FranklinTOliveira** [comment]
...but there is a catch!

I still need to make the Java for KingsTableRows produce a line of whitespace for the null string (the Java for HouseInformationPage produces an apology). Right now, running the site through the browser produces a RuntimeException.

File: src/main/java/website/fgto/royals_of_britain/view/houses/HouseInformationPage.html [hunk 1]
```diff
@@ -38,7 +38,8 @@
                 <div id="intro" class="row"  wicket:id="introPanel">
                     <p>King Franklin III was born in the County of Sacramento.</p>
                 </div>
-                <div id="table" class="row">
+                <!--This is NoTable, an adjustment to have this web page not have a table, if without parameters.-->
```
Comment (kyleoliveira) [inline src/main/java/website/fgto/royals_of_britain/view/houses/HouseInformationPage.html:41]: Gotta remove this comment

File: src/main/java/website/fgto/royals_of_britain/view/kings/KingsTable.java [hunk 2]
```diff
@@ -8,31 +8,33 @@
...
-
-            row.add(new Label("kingsName", new Model<>(king.getName())));
-            row.add(new Label("birthYear", new Model<>(king.getYearOfBirth())));
-            row.add(new Label("deathYear", new Model<>(king.getYearOfDeath())));
-
-            rows.add(row);
-          }
-      );
-    return rows;
+      kingsService
```
Comment (kyleoliveira) [inline src/main/java/website/fgto/royals_of_britain/view/kings/KingsTable.java:24]: Seems like the only change for the lines in this method (i.e. 24-38 in the update) is some extra indentations. You should change it back to what it was before.

**Pull request merged**